### ISSUE TITLE
Old version of Support Library, request update

### ISSUE THREAD

**Issue by mateiradu** [roughike/BottomBar#784]
### TL;DR
Bumping Support Library to the latest version will avoid mixing versions with other dependencies that already use version 25.3.1.

### Long version
Using BottomBar v2.3.1 forces me to use old Support Library dependencies:

> A newer version of com.android.support:support-v4 than 25.3.0 is available: 25.3.1

If I try to use the latest 25.3.1 Support Library my app will not build because of mixed versions since BottomBar uses the former release.

> All com.android.support libraries must use the exact same version specification (mixing versions can lead to runtime crashes). Found versions 25.3.1, 25.3.0

My workaround was to use the older version and keep the warnings but now I added the Facebook Android SDK that relies on 25.3.1 so this no longer works.

Updating BottomBar to use the latest Support Library will fix this issue.
